.github/workflows/chaos-test.yml

```
@@ -0,0 +1,112 @@
+          # Set Redis failure probability
+          PROBABILITY="${{ github.event.inputs.redis_failure_probability || '0.3' }}"
+          curl -X POST http://localhost:3000/chaos/set-probability \
+            -H "Content-Type: application/json" \
+            -H "Authorization: Bearer $TOKEN" \
+            -d "{\"probability\":$PROBABILITY}"
+          
+          # Run a series of API calls to test resilience
+          for i in {1..50}; do
+            # Test various endpoints under chaos conditions
+            curl -s -o /dev/null -w "%{http_code}" http://localhost:3000/health || true
+            
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)